distribution-core/pom.xml

```diff
@@ -462,26 +462,6 @@
             </activation>
             <build>
                 <plugins>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-antrun-plugin</artifactId>
-                        <version>${maven.antrun.version}</version>
```

Comment:
To be confirmed, but it should be possible to remove this property in the parent pom.xml

Reply:
removed